tests/unit/connect.c

```diff
@@ -49,16 +49,18 @@ int main(int argc, char *argv[])
 {
   (void)argc;
   (void)argv;
-
-  drizzle_options_st *opts = drizzle_options_create();
+  drizzle_options_st *opts;
```

Comment:
It was solely to define all variables at the top of the `main` function. But see that it is unnecessary given that the options are not used in the connection failure tests. 
Will revert this change when addressing the comments